### ISSUE TITLE
Framepool: "DO NOT USE"

### ISSUE THREAD

**Issue by mccalluc** [WGBH/stock-sales-admin#9]
0749-001-04-04-003 has a big "DO NOT USE": Should it not appear on the website? Could we drop it from the spreadsheet? (I feel like I saw another case like this somewhere, but can't locate it now...)

**Comment by lynnmason** [comment]
Yes.  Anything that says DO NOT USE or PLEASE REMOVE should not be included on the website.
